int16[] blocks_over
int16 gripper_at
bool gripper_closed


